views/lexicon/tabs/progress_bars.html.twig

```diff
@@ -216,5 +216,32 @@
         </a>
 
     </div>
+    <br/>
+    <h2 class="heading">Stepped Progress Bar</h2>
+
+    <div class="stepped-progressbar">
```

Comment:
I'm not 100% clear on how this will be used but do we need to consider aria `progressbar` role

Reply:
It is created for presentational purposes only, not navigation. Just presenting the current step the user is in a series of modals

Reply:
From https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_progressbar_role

> The progressbar role should be used for an element that displays the progress status for a task that take a long time or consists of several steps.

Reply:
Interesting information but still compatibility to be announced and W3C pages are 404